src/style.css

```diff
@@ -0,0 +1,115 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Book.ttf");
+  font-style: normal;
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Heavy.ttf");
+  font-style: normal;
+  font-weight: 900;
+}
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+  font-size: 16px;
+}
+
+.container {
+  padding: 20px 8px;
+}
+
+.form {
+  width: 100%;
+  position: relative;
+}
+
+.form:not(:last-child) {
+  margin-bottom: 20px;
+}
+
+.label {
+  display: flex;
+  align-items: center;
+  height: 100%;
+  padding: 0 17px 0 26px;
+  position: absolute;
```

Comment:
you should move `absolute` element with `top/bottom/right/left` properties, not paddings